pkg/pb/encryption.proto

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package encryption;
+
+import "gogo.proto";
+
+message EncryptionParameters {
+  CipherSuite cipher_suite = 1;
+  int64 block_size = 2;
+}
+
+message CipherSuite {
+  enum Type {
```

Comment:
This is Cipher, not CipherSuite

Reply:
ok i changed to Cipher then. Whats the difference? it looks like in our codebase we use both

Reply:
https://storjlabs.atlassian.net/browse/V3-2006

Reply:
Also there is a difference in what the enumerated values are.

Reply:
oh, egon is saying we *want* ciphersuite (and its value definitions) and we *don't want* cipher

Reply:
_updated_